src/Template/Pages/downloads.ctp

```diff
@@ -247,6 +250,34 @@ $transcriptionsOptions = $this->Downloads->createOptions('transcriptions');
             </dl>
         </div>
 
+        <!-- Sentences based on id  -->
+        <div  class="section md-whiteframe-1dp">
+            <?php /* @translators: section title in the Downloads page */ ?>
+            <h2><?= __('Sentences Based On ID') ?></h2>
+            <dl>
+                <dt><?= $filename ?></dt>
+                <?= $this->element(
+                    'per_language_files',
+                    [
+                        'model' => 'sentences',
+                        'options' => $sentencesBasedOnIdOptions
+                    ]
+                ) ?>
+                <dt><?= $description ?></dt>
+                <dd>
+                    <?= format(
+                        __(
+                            'Contains all the sentences in the selected language. '.
+                            'Each sentence is associated with a unique id and the id'.
+                            ' of the sentence it is based on.'
```

Comment:
It’s a bit more complicated actually. The value for `based_on_id` can be 3 different things:
* **a number greater than zero**: gives the sentence number it’s based on;
* **zero**: means the sentence is original, not based on any sentence;
* **`\N`** (which is how NULL is displayed in the CSV file): we don’t know. This is actually quite rare but we have a handful of sentences for which we don’t know whether they have been added as translations or not.